ansible/playbooks/Supporting_Scripts/Nagios_Ansible_Config_tool/templates/template.cfg

```diff
@@ -3,57 +3,42 @@ define host {
         host_name                       ReplaceHostName
         alias                           ReplaceAliasDescription
         address                         ReplaceIPAddress
+        check_command                   check_ssh!-4 -t 60
         max_check_attempts              5
         check_period                    24x7
         notification_interval           30
         notification_period             24x7
 	notifications_enabled           0
 }
-define service{
-        use                             local-service
-	host_name                       ReplaceHostName
-	service_description             SSH
-	check_command			check_ssh
-        }
-
-define service{
-        use                             local-service
-        host_name                       ReplaceHostName
-        service_description             PING
-	check_command			check_ping!200.0,20%!500.0,60%
-        }
 
 define service{
         use                             generic-service        
 	host_name                       ReplaceHostName
         service_description             Current Load
         check_command                   check_by_ssh!/usr/local/nagios/libexec/check_load -w 15,10,5 -c 30,24,20
-        }
-
-define service{
-        use                             generic-service        
-        host_name                       ReplaceHostName
-        service_description             Total Processes
-        check_command                   check_by_ssh!/usr/local/nagios/libexec/check_procs -w 170 -c 200
-        }
+        check_interval                  30
+}
 
 define service{
         use                             generic-service        
         host_name                       ReplaceHostName
-        service_description             Zombie Processes
-        check_command                   check_by_ssh!/usr/local/nagios/libexec/check_procs -w 5 -c 10 -s Z
-        }
+        service_description             Disk Space Root Partition
+        check_command                   check_by_ssh!/usr/local/nagios/libexec/check_disk -w 20% -c 10% -p /
+        check_period                    once-a-day-at-8
+}
 
 define service{
-        use                             generic-service        
+        use                             generic-service
         host_name                       ReplaceHostName
-        service_description             Current Users
-        check_command                   check_by_ssh!/usr/local/nagios/libexec/check_users -w 5 -c 10
-        }
+        service_description             Check Jenkins Connection
+        check_command                   check_agent!ReplaceHostName
+        check_period                    once-a-day-at-8
```

Comment:
I don't see a reason to do this only once a day at all. An agent should never be offline.